ui/components/config/meta-config-renderer.tsx

```
@@ -0,0 +1,193 @@
+                  </p>
+                </TooltipContent>
+              </Tooltip>
+            </TooltipProvider>
+          </CardTitle>
+        </CardHeader>
+        <CardContent className="space-y-4 px-0">
+          {config.fields.map((field) => (
+            <div key={field.name}>
+              <label className="text-sm font-medium block mb-2">
+                {field.label}
+              </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)